src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -205,7 +205,7 @@ private void LogSuccessfulTelemetryToClient(AuthenticationResult authenticationR
             }
         }
 
-        private Tuple<string, string> ParseScopesForTelemetry()
+        private (string resource, string scopes) ParseScopesForTelemetry()
```

Comment:
nit: I think we should just return (string, string) and deconstruct in the caller.

Reply:
naming them here informs the caller of what they are. rather than guessing